### PR TITLE
Makes solars auto-track again

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This PR fixes the issue of the tcomms solar tracker not auto-tracking the solars, and the AI having to either set them, or forget about it. The issue was the order of initializations, as visualised below
image

Now the solars initialize as part of the Sun subsystem (handles solar rotation and tracking, so I guess it makes sense)

## Why It's Good For The Game
The solars which have the auto-track flag in the code, should actually auto track instead of just being broken, redundant code.

## Changelog
:cl:
fix: Telecoms solars now auto-track again
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Well then, turns out this has no open issues for it.

**Comment by Spacemanspark** [comment]
Thank fuck

File: code/modules/power/solar.dm [hunk 1]
```diff
@@ -285,13 +285,14 @@
 	autostart = 1 // Automatically start
 
 /obj/machinery/power/solar_control/Initialize()
-	..()
-	if(!powernet)
-		return
+	SSsun.solars |= src
+	. = ..()
+
+/obj/machinery/power/solar_control/proc/setup()
```
Comment (Crazylemon64) [inline code/modules/power/solar.dm:291]: How would this work for maploaded solars? I don't see a mechanism for calling setup outside of the sun subsystem startup

Comment (AffectedArc07) [reply code/modules/power/solar.dm:291]: My bad

**Pull request merged**